examples/BUILD

```diff
@@ -28,6 +28,7 @@ filegroup(
         "jaeger-tracing/front-envoy-jaeger.yaml",
         "jaeger-tracing/service1-envoy-jaeger.yaml",
         "jaeger-tracing/service2-envoy-jaeger.yaml",
+        "lua-file/envoy.yaml",
```

Comment:
can you add your .lua file as a dependency of this file-group, or of the build rule that depends on it?

See other uses of data files in tests, e.g. the filesystem tests in test/common/runtime/BUILD